Use sm credentials in E2E tests (#2646)
kyma-project/kyma-environment-broker: docs/contributor/01-05-local-installation.md

   }'
   ```

> 4. Create a kubeconfig secret for the runtime. This simulates the kubeconfig that would normally be created by Kyma Infrastructure Manager (KIM):

Reviewer: ```suggestion
4. To simulate the kubeconfig created by Kyma Infrastructure Manager (KIM), create a kubeconfig Secret for the runtime.
```
